lattices/src/map_union_with_tombstones.rs

```diff
@@ -1,4 +1,41 @@
 //! Module containing the [`MapUnionWithTombstones`] lattice and aliases for different datastructures.
+//!
+//! # Choosing a Tombstone Implementation
```

Comment:
This would be moved into the module for the tombstone trait so that it is deduped with the set lattice.

Reply:
Great!